pilot/cmd/pilot-agent/main.go

```diff
@@ -752,6 +763,8 @@ func init() {
 	proxyCmd.PersistentFlags().StringVar(&mixerIdentity, "mixerIdentity", "",
 		"The identity used as the suffix for mixer's spiffe SAN. This would only be used by pilot all other proxy would get this value from pilot")
 
+	proxyCmd.PersistentFlags().StringVar(&meshConfigFile, "meshConfig", "/etc/istio/config/mesh",
+		"File name for Istio mesh configuration. If not specified, a default mesh will be used. MESH_CONFIG environment variable takes precedence.")
```

Comment:
nit:  If not specified, a default mesh will be used - is not clear. Do you mean to say `a default config will be used` ?

Reply:
s/"/etc/istio/config/mesh"/"" ?